src/components/enumFaces/enumFaces.tsx

```diff
@@ -0,0 +1,44 @@
+export enum SelectorType {
+  Carousel = "carousel",
+  Dropdown = "dropdown",
+  Multiselect = "multiselect",
+  Checkbox = "checkbox",
+  Radial = "radial",
+  Textinput = "textinput",
+}
+
+export enum Direction {
+  Left = "left",
+  Right = "right",
+}
+
+export enum FormDataType {
+  Name = "sandwichName",
+  Cutlery = "cutlery",
+  Napkins = "napkins",
+  Bread = "bread",
+  Cheese = "cheese",
+  Meat = "meat",
+  Dressing = "dressing",
+  Vegetables = "vegetables",
+  Egg = "egg",
+  Spreads = "spreads",
+  Serving = "serving",
+  Topping = "topping",
+}
+
+export interface PaniniFormData {
+  sandwichName: string[];
```

Comment:
**question:** why sandwich name is an array of strings?

Reply:
No real reason to be honest. Previously PaniniFormData was already formatted just like the payload, which was causing problems. I got frustrated so I made the object flat and changed all types to string arrays. I could change sandwichName to a single string, but it just looks consistent. ;)